proxy/envoy/mixer.go

```diff
@@ -103,3 +102,14 @@ func mixerHTTPRouteConfig(role proxy.Node) *FilterMixerConfig {
 		QuotaName: MixerRequestCount,
 	}
 }
+
+// Mixer TCP filter config for inbound requests
+func mixerTCPConfig(role proxy.Node) *FilterMixerConfig {
+	return &FilterMixerConfig{
+		MixerAttributes: map[string]string{
+			AttrTargetIP:  role.IPAddress,
+			AttrTargetUID: "kubernetes://" + role.ID,
+		},
+		QuotaName: MixerRequestCount,
```

Comment:
I think we decided only to enforce rate limits on number of new connections for TCP (but even that seems like it could come later). I don't think we want `MixerRequestCount` as the quota checked though.